### PR TITLE
LIVE-545 Add feature flag for enabling USB firmware update on LLM

### PR DESCRIPTION
**Pull request by grsoares21**
## Context (issues, jira)
[LIVE-545]

## Description / Usage
This branch creates the definition of a new feature flag to be used for the mobile firmware update feature.
It also puts string as a possible type for feature flags in order to allow us to develop even without upgrading live-common.

## Expectations

- [ ] **Test coverage: The changes of this PR are covered by test.** Unit test were added with mocks when depending on a backend/device.
- [x] **No impact: The changes of this PR have ZERO impact on the userland.** Meaning, we can use these changes without modifying LLD/LLM at all. It will be a "noop" and the maintainers will be able to bump it without changing anything.

[LIVE-545]: https://ledgerhq.atlassian.net/browse/LIVE-545?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by thomasrogerlux — APPROVED** [review]
I have mixed feelings about the `string` addition but okay

File: src/families/bitcoin/wallet-btc/explorer/index.ts [hunk 1]
```diff
@@ -207,28 +207,29 @@ class BitcoinLikeExplorer extends EventEmitter implements IExplorer {
...
-      await client.client.get(url, {
-        params,
-        // some altcoin may have outputs with values > MAX_SAFE_INTEGER
-        transformResponse: (string) =>
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          JSONBigNumber.parse(string, (key: string, value: any) => {
-            if (BigNumber.isBigNumber(value)) {
-              if (key === "value") {
-                return value.toString();
+    let res: { txs: TX[] } = { txs: [] };
```
Comment (gre) [inline src/families/bitcoin/wallet-btc/explorer/index.ts:210]: why change in this file? 🤔

Comment (grsoares21) [reply src/families/bitcoin/wallet-btc/explorer/index.ts]: Rebased and the change disappeared, should be fine now

Comment (grsoares21) [reply src/families/bitcoin/wallet-btc/explorer/index.ts]: Rebased and the change disappeared, should be fine now

**Pull request merged**